integration_tests/commands/async/hmget_test.go

```diff
@@ -0,0 +1,102 @@
+package async
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestHMGET(t *testing.T) {
+	conn := getLocalConnection()
+	var smallHashKeys []string
+	var smallHashValues []string
+
+	var bigHashKeys []string
+	var bigHashValues []string
+
+	// Setup smallHash
+	for i := 0; i < 8; i++ {
+		key := fmt.Sprintf("field%d", i)
+		value := fmt.Sprintf("value%d", i)
+		smallHashKeys = append(smallHashKeys, key)
+		smallHashValues = append(smallHashValues, value)
+		FireCommand(conn, "HSET smallHash "+key+" "+value)
+	}
+
+	// Setup bigHash
+	for i := 0; i < 1024; i++ {
+		key := fmt.Sprintf("field%d", i)
+		value := fmt.Sprintf("value%d", i)
+		bigHashKeys = append(bigHashKeys, key)
+		bigHashValues = append(bigHashValues, value)
+		FireCommand(conn, "HSET bigHash "+key+" "+value)
+	}
+
+	defer conn.Close()
+	defer FireCommand(conn, "DEL key_hmGet key_hmGet1 smallHash bigHash")
+
+	testCases := []TestCase{
+		{
```

Comment:
nit: adding name field in the test cases gives future collaborators a concise idea about what the test case is validating.

Reply:
Thanks for suggesting this as well, I've added the names for the tests.